src/c/config.c

```diff
@@ -312,10 +311,8 @@ void edgex_device_parseTomlClients
 {
   if (clients)
   {
-    parseClient (lc, toml_table_in (clients, "core-data"), &endpoints->data, err);
```

Comment:
If we want to get all readings for a given device, as a starting point before listening to the message bus, we normally use the REST API to request this (with paging and all). This is what core caching is going to make better.
Is there a Message Bus way to request this data?
Otherwise, we'll still want to keep this client configuration.
"We" here are app services using the device SDK - because "everything is becoming a device service".

Reply:
Think the answer is that we'll keep the REST client config for now

Reply:
Updated. Also restored the 'ping' of core-data at startup which is also done on REST rather than MessageBus

Reply:
Actually that's not right. The device service generally doesn't require core-data to be running, but @TomBrennan-Eaton in your application it does? Do we need to expose from the SDK the ability to ping core-data?

Reply:
I've taken the core-data related code back out, but left the configurations in. @TomBrennan-Eaton does #441 cover your needs?

Reply:
OK, I'm satisfied. And #441 looks like just what we'll need. (I was going to ask about a client for core-command next...)